From 849771b2e72d10802c8a8028282514b873b61c2f Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Tue, 27 Jan 2026 15:18:42 +0100
Subject: [PATCH 1/2] disable tests

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1da7c8f747..30e5847423 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.22)
 message(STATUS "CMake version: ${CMAKE_VERSION}")
 
+set(BUILD_TESTING OFF)
+
 # Set up project properties
 set(PROJECT_NAMESPACE loco-3d)
 set(PROJECT_NAME crocoddyl)
-- 
2.53.0

